application/src/main/scala/com/azavea/franklin/api/Server.scala

```diff
@@ -134,10 +142,8 @@ $$$$
         ).routes
         landingPageRoutes = new LandingPageService[IO](apiConfig).routes
         router = CORS(
-          Router(
```

Comment:
`Router` can be used to map different services on different endpoints, but `tapir` is already doing that for us with the endpoint definitions / http4s interpreter, so we can unindent